draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1536,229 +1485,97 @@ easy enough to parse with existing implementations (i.e., by splitting the file
-definitions to [the designated public
-repository](https://github.com/quiclog/qlog).
-
-The following table provides an overview of all the discussed qlog formatting
-options with examples:
-
-| format                                    | qlog_format               | extension        | media type                  |
-|-------------------------------------------|---------------------------|------------------|-----------------------------|
-| JSON {{format-json}}                      | JSON                      | .qlog            | application/qlog+json       |
-| JSON Text Sequences  {{format-json-seq}}  | JSON-SEQ                  | .sqlog           | application/qlog+json-seq   |
-| named headers {{structure-optimizations}} | JSON(-SEQ).namedheaders   | .(s)qlog         | application/qlog+json(-seq) |
-| dictionary {{structure-optimizations}}    | JSON(-SEQ).dictionary     | .(s)qlog         | application/qlog+json(-seq) |
-| CBOR {{binary}}                           | JSON(-SEQ)                | .(s)qlog.cbor    | application/qlog+json(-seq)+cbor(-seq) |
-| protobuf {{binary}}                       | protobuf                  | .qlog.protobuf   | NOT SPECIFIED BY IANA       |
-|                                           |                           |                  |
-| gzip {{compression}}                      | no change                 | .gz suffix       | application/gzip            |
-| brotli {{compression}}                    | no change                 | .br suffix       | NOT SPECIFIED BY IANA       |
-
-## Conversion between formats {#conversion}
-
-As discussed in the previous sections, a qlog file can be serialized in a
-multitude of formats, each of which can conceivably be transformed into or from
-one another without loss of information. For example, a number of JSON-SEQ
-streamed qlogs could be combined into a JSON formatted qlog for later processing.
-Similarly, a captured binary qlog could be transformed to JSON for easier
-interpretation and sharing.
+Formal definition of additional qlog formats or encodings that use the
+optimization techniques described here, or any other optimization technique is
+left to future activity that can apply the following guidelines.
 
-Secondly, we can also consider other structured logging approaches that contain
-similar (though typically not identical) data to qlog, like raw packet capture
-files (for example .pcap files from tcpdump) or endpoint-specific logging formats
-(for example the NetLog format in Google Chrome). These are sometimes the only
-options, if an implementation cannot or will not support direct qlog output for
-any reason, but does provide other internal or external (e.g., SSLKEYLOGFILE
-export to allow decryption of packet captures) logging options For this second
-category, a (partial) transformation from/to qlog can also be defined.
+It is important that tooling can correctly parse and process serialized qlog. It
```

Comment:
This first sentence seems a bit... obvious. Did this get lost in "translation"? What are we actually trying to say? Process serialized JSON / I-JSON / JSON-SEQ qlog?

Reply:
fixed!